packages/rancher-gatekeeper/overlay/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+All notable changes from the upstream OPA Gatekeeper chart will be added to this file
+
+## [Package Version 00] - 2020-07-27
+### Added
+- Enabled the CRD chart generator in `package.yaml`
+
+### Modified
+- Updated chart version in `Chart.yaml` to follow the upstream's format `v3.1.0-beta.X`
+- Updated namespace to `cattle-gatekeeper-system`
+- Updated `rancher/istio-kubectl` image to `1.5.8`
+- Updated for Helm 3 compatibility
+    - Moved crds to `crds` directory
+    - Removed `crd-install` hooks and templates from crds
+
+### Removed
+- Removed the following files as the `gatekeeper-webhook-service` was removed in our previous version of the chart
+    - `gatekeeper-validating-webhook-configuration-validatingwebhookconfiguration.yaml`
+    - `gatekeeper-webhook-service-service.yaml`
+- Removed `gatekeeper-system-namespace.yaml` as Rancher handles namespaces for chart installation
+- Removed unnecessary `index.yaml` as we package and host our charts
```

Comment:
Good changelog, I think the modified/removed sections are valuable to end users.